### PR TITLE
fix(tests): Migrate from manifests to helm in CI tests

### LINKED ISSUE #1269: bug(syscalls): not receiving events for audited syscalls
NOTE: This bug is also causing frequent failures in ginkgo tests for docker.
## Bug Report

**General Information**

- Environment description: k3s with docker
	```
	vagrant@kubearmor-dev-next:~/accuknox/KubeArmor/tests$ k3s --version
	k3s version v1.23.9+k3s1 (f45cf326)
	go version go1.17.5
	vagrant@kubearmor-dev-next:~/accuknox/KubeArmor/tests$ docker --version
	Docker version 20.10.13, build a224086
	```
- Kernel version: `Linux kubearmor-dev-next 5.15.0-46-generic #49-Ubuntu SMP Thu Aug 4 18:03:25 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux`
- Orchestration system version in use:
	```
	Server Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.9+k3s1", GitCommit:"f45cf3267307b153ed8b418ae5b8ea6c6b9ebaca", GitTreeState:"clean", BuildDate:"2022-07-19T00:42:17Z", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}

	```
- Link to relevant artifacts (policies, deployments scripts, ...)
- Target containers/pods

**To Reproduce**

1. Deploy kubearmor using https://github.com/kubearmor/KubeArmor/blob/main/KubeArmor/build/kubearmor-test-docker.yaml - the kubearmor args in this are a possible cause.
2. Try to generate some events. Generally running ksp test suite should be fine.
	```
	cd tests/
	ginkgo --flake-attempts=5 ksp/ syscalls/
	```
3. In the above, `syscall` test suite might fail. If it doesn't, run `karmor logs --gRPC=:32767 --operation=syscall --logFilter=all`, try to exec into one of the multiubuntu pods created by the above and run:
	```
	cp `which unlink` /unlink
	touch /dummy
	unlink /dummy
	``` 
4. You won't get a log for syscall=SYS_UNLINK. However, other syscall logs like SETUID, SETGID would be working fine.

**Expected behavior**

Should receive logs for audited syscalls.

### PR DESCRIPTION
**Pull request by DelusionalOptimist**
**Purpose of PR?**:

Possibly fixes #1269 

**Does this PR introduce a breaking change?**
No

**If the changes in this PR are manually verified, list down the scenarios covered:**:
`make -C tests/`

**Additional information for reviewer?** :
Based on the analysis done in #1269, audited syscall events generally get dropped only after we have received a large number of events. It is strange as other events still seem to come by. Since reproducing this takes time and we want to have a stable CI in the upcoming release, this is a proposed quickfix which attempts to fix failures by:
* Running syscalls suite first
* Deleting deployments created by test suite once it completes
* Migrating test manifests to helm chart

**Checklist:**
- [x] Bug fix. Fixes #<issue number>
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: .github/workflows/ci-test-controllers.yml [hunk 1]
```diff
@@ -1,31 +1,61 @@
 name: ci-test-controllers
 
 on:
-  push:
```
Comment (daemon1024) [inline .github/workflows/ci-test-controllers.yml:4]: why not test on main?

Comment (daemon1024) [inline .github/workflows/ci-test-controllers.yml:4]: Why not test on main?

**Review by daemon1024 — APPROVED** [review]
LGTM! One query inline

File: .github/workflows/ci-test-controllers.yml (hunk 1 above)
Comment (DelusionalOptimist) [reply .github/workflows/ci-test-controllers.yml:4]: It runs the ginkgo test. Just seemed redundant given that we already test it on PR and we have ginkgo tests running in other test suites. Also, the build tests are still run on main.

**Pull request merged**